index.bs

```diff
@@ -60,6 +60,9 @@ spec: WEBDRIVER; urlPrefix: https://w3c.github.io/webdriver/
     text: WebDriver new session algorithm; url: dfn-webdriver-new-session-algorithm
     text: web element reference; url: dfn-web-element-reference
     text: window handle; url: dfn-window-handle
+spec: CONSOLE; urlPrefix: https://console.spec.whatwg.org
```

Comment:
Want to file an issue about integration hooks for this?

Reply:
https://github.com/whatwg/console/issues/195